contrib/node/src/python/pants/contrib/node/subsystems/node_distribution.py

```diff
@@ -45,17 +48,29 @@ def register_options(cls, register):
                  NodeDistribution.VALID_PACKAGE_MANAGER_LIST.keys()))
       register('--yarnpkg-version', advanced=True, default='v0.19.1', fingerprint=True,
                help='Yarnpkg version. Used for binary utils')
+      register('--eslint-setupdir', advanced=True, fingerprint=True,
```

Comment:
It seems like all of these options should potentially be on an "eslint" subsystem instead, which could depend on the `NodeDistribution` subsystem?

Also, I don't think have an explicit `setupdir` option is a good idea, as you could always just do it in a temp directory, right?

Reply:
The setup dir here should not be in a temp directory, it should be a directory that gets checked in and contains a package.json that configures "eslint".